app/experimenter/experiments/models/nimbus.py

```diff
@@ -179,7 +178,7 @@ def targeting_config(self):
             return self.TARGETING_CONFIGS[self.targeting_config_slug]
 
     def latest_change(self):
-        return self.changes.order_by("-changed_on").first()
+        return self.changes.latest_change()
```

Comment:
Oh maybe this whole method is redundant and we should just replace it with `experiment.changes.latest_change()` wherever it's called?  NBD we can do that as cleanup later